### ISSUE TITLE
Putting charcoal dust in pitch container causes tesselation error in 1.17.0-rc.5

### ISSUE THREAD

**Issue by JnJNyarly** [TaskaRaine/Ancient-Tools#48]
```Running on 64 bit Windows with 64 GB RAM 
Game Version: v1.17.0-rc.5 (Unstable)
Loaded Mods: acorns@0.5.1, weaponpacka@1.2.4, ancienttools@1.4.5, backpackpackplus@1.0.3, bettercrates@1.4.2, craftables@1.1.1, cralutweaks@1.0.2, cruciblecapacity@1.0.1, fixroads@1.3.1, geothermalinsanity@1.1.0, HangingOilLamps@1.0.1, hideandfabric@1.2.5, aaamakeitcompatible@0.0.2, meteoricexpansion@1.2.1, moreclasses@1.2.4, moredrifterloot@1.5.4, MoreResin@1.0.0, moreroads@1.4.3, MoreTorchHolders@1.0.0, necessaries@1.1.2, nodrifterthrowing@1.0.0, orepulverizer@1.1.1, OresAPlenty@2.5.0, primitivesurvival@3.0.3, qptech@1.10.3-pre.5, rope-bridges@1.1.0, riflemod@1.0.0, superfuel@2.0.7, viescraftmachines@1.17.2, game@1.17.0-rc.5, zoombuttonreborn@1.3.0, egocaribautomapmarkers@2.4.0-pre.1, carrycapacity@0.6.5, extraoverlays@1.1.0, fixhandbookclutter@1.0.5, immersivecrafting@1.0.0, jhome@1.0.5, miningtools@1.1.4, petai@1.5.2, paw@1.0.2, primitivetools@1.3.1, prospectorinfo@3.2.0, resinlightsbymad@1.0.1, smeltableingots@1.0.0, techmolds@1.0.0, creative@1.17.0-rc.5, survival@1.17.0-rc.5, bullseye@2.4.0-pre.2, cats@1.5.1, darkerspawns@1.0.0, defensive@0.1.6, foxtaming@1.0.1, moremolds@1.2.0, wolftaming@1.5.2
2022-08-11 13:23:54: Critical error occurred
System.MissingMethodException: Method not found: 'Void Vintagestory.API.Client.ITesselatorAPI.TesselateShape(System.String, Vintagestory.API.Common.Shape, Vintagestory.API.Client.MeshData ByRef, Vintagestory.API.Client.ITexPositionSource, Vintagestory.API.MathTools.Vec3f, Int32, Byte, Byte, System.Nullable`1<Int32>, System.String[], Int32)'.
   at AncientTools.Utility.DisplayInventory.GenMesh(ICoreClientAPI capi, String shapePath)
   at AncientTools.BlockEntities.BEPitchContainer.UpdateMeshes() in C:\Projects\vsmods\mods\ancienttools\src\blockentity\pitch\BEPitchContainer.cs:line 103
   at AncientTools.BlockEntities.BEPitchContainer.InsertObject(ItemSlot playerActiveSlot, ItemSlot inventorySlot, Int32 takeQuantity) in C:\Projects\vsmods\mods\ancienttools\src\blockentity\pitch\BEPitchContainer.cs:line 162
   at AncientTools.BlockEntities.BEPitchContainer.OnInteract(IPlayer byPlayer) in C:\Projects\vsmods\mods\ancienttools\src\blockentity\pitch\BEPitchContainer.cs:line 153
   at AncientTools.Blocks.BlockPitchContainer.OnBlockInteractStart(IWorldAccessor world, IPlayer byPlayer, BlockSelection blockSel) in C:\Projects\vsmods\mods\ancienttools\src\blocks\pitch\PitchContainer.cs:line 207
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.TryBeginUseBlock(Block selectedBlock, BlockSelection blockSelection) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\Systems\Player\MouseInWorldInteractions.cs:line 890
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.HandleMouseInteractionsBlockSelected(Single dt) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\Systems\Player\MouseInWorldInteractions.cs:line 536
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnFinalizeFrame(Single dt) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\Systems\Player\MouseInWorldInteractions.cs:line 89
   at Vintagestory.Client.NoObf.ClientEventManager.TriggerRenderStage(EnumRenderStage stage, Single dt) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\Util\ClientEventManager.cs:line 192
   at Vintagestory.Client.NoObf.ClientMain.TriggerRenderStage(EnumRenderStage stage, Single dt) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\ClientMain.cs:line 1176
   at Vintagestory.Client.NoObf.ClientMain.RenderToDefaultFramebuffer(Single dt) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\ClientMain.cs:line 1364
   at Vintagestory.Client.GuiScreenRunningGame.RenderToDefaultFramebuffer(Single dt) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\MainMenu\Screens\GuiScreenRunningGame.cs:line 204
   at Vintagestory.Client.ScreenManager.Render(Single dt) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\ScreenManager.cs:line 706
   at Vintagestory.Client.ScreenManager.OnNewFrame(Single dt) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\ScreenManager.cs:line 610
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\ClientPlatform\GameWindow.cs:line 132
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp) in C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:line 479
   at OpenTK.GameWindow.DispatchRenderFrame() in C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:line 454
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second) in C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:line 375
   at Vintagestory.Client.ClientProgram.Start(ClientProgramArgs args, String[] rawArgs)
   at Vintagestory.ClientNative.CrashReporter.Start(ThreadStart start) in C:\Users\Tyron\Documents\vintagestory\game\VintagestoryLib\Client\ClientPlatform\ClientNative\CrashReporter.cs:line 87
-------------------------------
```

**Comment by TaskaRaine** [comment]
That's strange... the method still does exist with the rc-5 API. It seems that building the mod with rc-5 was enough to fix the issue so this should be solved with the next release. 

Thanks for bringing this to my attention!

**Comment by TaskaRaine** [comment]
Should be solved in version 1.4.6
